lib/pages/chat_draft/draft_chat_view.dart

```diff
@@ -67,196 +67,210 @@ class DraftChatView extends StatelessWidget {
+                                            animation: primaryAnimation,
+                                            secondaryAnimation:
+                                                secondaryAnimation,
+                                            transitionType:
+                                                SharedAxisTransitionType.scaled,
+                                            fillColor: Colors.transparent,
+                                            child: child,
+                                          );
+                                        },
+                                        child: !controller.showEmojiPicker
+                                            ? TwakeIconButton(
+                                                paddingAll:
+                                                    controller.inputText.isEmpty
+                                                        ? 5.0
+                                                        : 12,
+                                                tooltip:
+                                                    L10n.of(context)!.emojis,
+                                                onPressed: () => controller
+                                                    .emojiPickerAction(),
+                                                icon: Icons.tag_faces,
+                                              )
+                                            : TwakeIconButton(
+                                                paddingAll:
+                                                    controller.inputText.isEmpty
+                                                        ? 5.0
+                                                        : 12,
+                                                tooltip:
+                                                    L10n.of(context)!.keyboard,
+                                                onPressed: () => controller
+                                                    .inputFocus
+                                                    .requestFocus(),
+                                                icon: Icons.keyboard,
+                                              ),
+                                      ),
+                                    ),
                                   ),
-                                ),
+                                  if (PlatformInfos.platformCanRecord &&
+                                      controller.inputText.isEmpty)
+                                    Container(
+                                      height: 56,
+                                      alignment: Alignment.center,
+                                      child: TwakeIconButton(
+                                        margin:
+                                            const EdgeInsets.only(right: 7.0),
+                                        paddingAll: 5.0,
+                                        onPressed: () {},
```

Comment:
I think should show dialog: `Feature not implemented yet`

Reply:
Currently we disable this feature. I'll update it in another PR